src/wp-content/themes/twentytwentythree/readme.txt

```diff
@@ -9,11 +9,9 @@ License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
 == Description ==
 
-Twenty Twenty-Three is a diverse, collaborative effort designed with the expansion of the possibilities of the block editor in mind. With a blank, unopinionated base as a starting point, this year’s default theme has minimal demo content to get in your way.
```

Comment:
Unfortunately, I think that these text changes will need to wait until after 6.1 is released.

RC1 marked the start of [hard string freeze](https://make.wordpress.org/polyglots/handbook/glossary/#hard-freeze). If these are absolutely necessary, we would need to coordinate with the Polyglots team and make them aware as the new strings will need to be re-translated. This could result in translations not being available on release day.

Where this is the main description of the theme, I think I lean towards not changing it so that it will be available in more languages initially.